uniswap_swap_detection/src/agent.spec.ts

```diff
@@ -0,0 +1,244 @@
+import {
+  FindingType,
+  FindingSeverity,
+  Finding,
+  HandleTransaction,
+  ethers,
+} from "forta-agent";
+import { utils, Contract } from "ethers";
+
+import agent, { provideHandleTransaction } from "./agent";
+import { SWAP_EVENT } from "./constant";
+import { createAddress } from "forta-agent-tools";
+import {
+  MockEthersProvider,
+  TestTransactionEvent,
+} from "forta-agent-tools/lib/test";
+
+const iface: utils.Interface = new utils.Interface([
+  "function token1() external view returns (address)",
+  "function token0() external view returns (address)",
+  "function fee() external view returns (uint24)",
+]);
+
+export const CREATED_POOL =
```

Comment:
```suggestion
export const poolCreatedEvent =
```
Small suggestion so it's more clear when in use. Also, the team has a practice on not really naming test variables in all caps like they would be in production.